dandischema/models.py

```diff
@@ -1484,6 +1484,11 @@ class PublishedDandiset(Dandiset, Publishable):
         description="Permalink to the Dandiset.",
         nskey="schema",
     )
+    releaseNotes: Optional[str] = Field(
+        None,
+        description="User notes for describing the release",
```

Comment:
minor suggestion -- to shorten to

```suggestion
        description="Description of the release",
```
since could be a proud "Data curator" who submits the notes etc ;)

Reply:
FWIW, it is minor but I think we better use "my" version since would be consistent with other similar ones
```
dandischema/models.py:        None, description="The description of the equipment.", nskey="schema"
dandischema/models.py:        None, description="The description of the activity.", nskey="schema"
dandischema/models.py:        None, description="A brief description of the project.", nskey="schema"
dandischema/models.py:        None, description="A brief description of the session.", nskey="schema"
dandischema/models.py:        description="Genotype descriptor of participant or subject if available",
dandischema/models.py:        None, description="A description of the item.", nskey="schema"

```

since indeed could be not a "user" but someone else etc.